Material.DataGrid/Material.DataGrid.csproj

```diff
@@ -21,8 +21,8 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Avalonia" Version="0.10.11" />
-	  <PackageReference Include="Avalonia.Controls.DataGrid" Version="0.10.11" />
+      <PackageReference Include="Avalonia" Version="0.10.13" />
+	  <PackageReference Include="Avalonia.Controls.DataGrid" Version="0.10.13" />
```

Comment:
Have some strange extra spacing in this line
not such critical point but recommend remove it to keep code clean